pkg/github/hint_comment_service.go

```diff
@@ -0,0 +1,81 @@
+package github
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/arquillian/ike-prow-plugins/pkg/log"
+	"github.com/arquillian/ike-prow-plugins/pkg/scm"
+	"github.com/arquillian/ike-prow-plugins/pkg/utils"
+)
+
+// PluginTitleTemplate is a constant template containing "Ike Plugins (name-of-plugin)" title with markdown formatting
+const (
+	PluginTitleTemplate     = "### Ike Plugins (%s)"
+	assigneeMentionTemplate = "Thank you @%s for this contribution!"
+)
+
+// HintCommentService is a struct managing plugin comments
+type HintCommentService struct {
```

Comment:
I think simple `HintService` (or even `Hinter`) would do too.

Reply:
`Hinter` is cool :+1: